### DISCUSSION TITLE
Is Windows ARM64 supported?

### DISCUSSION THREAD

**Discussion by infinitepower18** [pyinstaller/pyinstaller#7188] [PyInstaller] [answered]
Python 3.11 is the first version to officially support ARM64 Windows. I have tried to build an exe with a simple test program but it crashes when I try to run it. However it works fine if I use the AMD64 Python. So is ARM64 supported by Pyinstaller currently?

**Comment by bwoodsend** [comment]
No it's not. We need a volunteer with an arm64 windows machine to figure out what needs to go [here](https://github.com/pyinstaller/pyinstaller/blob/780c08d7f1827b37bfc94751a411d71451e04a35/bootloader/wscript#L331-L340).

**Reply by infinitepower18** [reply]
Not sure if this will help but if you have an Apple Silicon Mac you can install Windows 11 ARM64 using Parallels or UTM (free software) and then run the ARM64 Windows version of Python on it.

**Reply by bwoodsend** [reply]
Huh, if I'd known that, I'd have done it by now. I have to use one of those miserable M1 machines for work. I'll see if I can get it working...

**Comment by infinitepower18** [comment, marked as answer]
Windows ARM64 support has been added in v5.7.0. Just tested it and it's working perfectly.
